src/main/java/yanagishima/config/WebMvcConfig.java

```diff
@@ -0,0 +1,17 @@
+package yanagishima.config;
+
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.servlet.config.annotation.DefaultServletHandlerConfigurer;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurerAdapter;
+
+@Configuration
+@EnableWebMvc
+@ComponentScan(basePackages = { "yanagishima.controller" })
```

Comment:
`yanagishima.controller` package doesn't exist now.
Will you move XXXServlet to `yanagishima.controller`?

Reply:
Yes, I will move to the package and rename to XXXController later. I avoided it for minimizing diff in this PR.